docs/Android/allowed-apps.md

```diff
@@ -0,0 +1,76 @@
+
+For more on this API, visit: [setSDKControlConfig](crittercism.md#setsdkcontrolconfigconfig)
+
+
+```Java
+/**
+* Sets the WS1 UEM SDK Control Config.
+* IntelligenceSDK will parse for the following properties:
+*  - IntelSDKAllowedApps: JSON Array of apps allowed to transmit DEX data to the UEM console. If not available,
+*    IntelligenceSDK will default to allowing all apps to transmit data.
+*  - DEXData: JSON containing the privacy configuration.
+*  Apps / SDK's are expected to fetch the control config JSON string and call this API. In the case the control
+*  config is not available, a null value should be passed for the config parameter.
+*
+* @param config Json String of the control config provided from the WS1 UEM SDK.
+*/
+public static void setSDKControlConfig(String config)
+```
+
+
+### Control Configuration Attributes
+The following is a table of the Attributes that are currently parsed within the control config.
+
+#### Configuration JSON Attributes Table
+|         Flag        |                                       Description                                       |   Value   |                                                   Outcome                                                   |                                      Notes                                     |
+|:-------------------:|:---------------------------------------------------------------------------------------:|:---------:|:-----------------------------------------------------------------------------------------------------------:|:------------------------------------------------------------------------------:|
+| IntelSDKAllowedApps | JSON Array of Application IDs that are allowed to transmit DEX data to the UEM console. |           |                                                                                                             | This value of this key (JSON object) is parsed only if CaptureDEXData is true. |
+|                     |                                                                                         |  MISSING  |                             All apps will transmit DEX data to the UEM console.                             |                                                                                |
+|                     |                                                                                         | AVAILABLE | Only the matching Application IDs within the array will be allowed to transmit DEX data to the UEM console. |                                                                                |
+
+### Example Payload
+
+```JSON
+  // Only the listed Android apps (App 1 and App 2) will be allowed to transmit DEX data.
+
+  "IntelSDKAllowedApps": [
+    "2b5a662e8c4d45fe9f8af99af509655b00555300",  // App 1 ID
+    "84e3527ecb354764925d48345530c47f00555300"  // App 2 ID
+  ]
+```
+
+
+### All Attributes and Events Reported By IntelligenceSDK
```

Comment:
Can we remove this section and the two below this unless there is a reason to keep it?